package.json

```diff
@@ -48,7 +48,8 @@
     "eslint-plugin-prettier": "4.0.0",
     "lint-staged": "12.4.1",
     "prettier": "2.6.2",
-    "typescript": "^4.4.4"
+    "typescript": "^4.4.4",
+    "syncpack": "8.2.4"
```

Comment:
are we exploring syncpack?

Reply:
Hmmm not sure if i understand the question. We're currently using it to build `content` and `lambdas` with the same dependency versions. Just adding it to `devDependencies` so we don't have to install between every `yarn install, yarn build` cycle.